src/lib/netlist/plib/pstream.h

```diff
@@ -299,7 +299,7 @@ class pbinary_writer
 	template <typename T>
 	void write(const std::vector<T> &val)
 	{
-		const auto sz(val.size());
+		const T sz(val.size());
```

Comment:
This shouldn’t be `T` – it’s `std::vector<T>::size_type` (in practice it’s `std::size_t` unless `std::vector` is specialised for `T`).